### ISSUE TITLE
Remove/bypass depreciated function MagickGetImageAttribute

### ISSUE THREAD

**Issue by lsproc** [mkoppanen/imagick#78]
Hi,

MagickGetImageAttribute appears to have been depreciated and does not exist in the 6.8.9 or 6.9.0 DLLs and hence the extension as downloaded from PECL does not work with these versions.

**Comment by Danack** [comment]
Hi @lsproc,

So......the issue you're seeing is that Imagick needs to be compiled against the version of ImageMagick that it is going to be used with as during the build process of Imagick it looks to see if those functions should be compiled or not.

I think the only sane way to do this would be to ask the guys who maintain http://windows.php.net/ to start providing compiled versions of ImageMagick as well as Imagick.

This would solve not only your problem, but also be good as there have been many memory related issues fixed in the underlying ImageMagick library.

**Comment by lsproc** [comment]
Hi @Danack thanks for your reply.

I have investigated a bit further and it seems that ImageMagick developers have decided to change whether they build their Windows releases with depreciated functions enabled or not around version 6.8.8 (the last one I got it to work for was 6.8.7-8 should you wish to note this in the documentation).

Thank you for the advice though on building it myself - I will look into doing this.
